README.md

```diff
@@ -12,7 +12,13 @@ Therefore, ArgoCD is the main requirement to run this project on your cluster.
 
 ![Quick Start](docs/img/quick-start/k8s-addons-quick-start.png)
 
-Assuming you already have `argocd` (if installed with helm, the name of the chart should be argocd) deployed in your cluster to make deployment successful and all addons synced it is required to verify if following configuration is part of your ArgoCD configuration: [argocd/values.yaml](https://github.com/nearform/k8s-kurated-addons/blob/main/addons/argocd/values.yaml#L23).
+If you dont have `argocd`on your cluster, the following command will install it + `app-of-apps`, setting up everything. Make sure that you are using the correct Kubernetes context before run.
+
+```bash
+$ make cloud
+```
+
+If you already have `argocd` (if installed with helm, the name of the chart should be argocd) deployed in your cluster to make deployment successful and all addons synced it is required to verify if following configuration is part of your ArgoCD configuration: [argocd/values.yaml](https://github.com/nearform/k8s-kurated-addons/blob/main/addons/argocd/values.yaml#L23).
```

Comment:
why line 23 explicitly in the link?

Reply:
I dont know why, it is already there.

Should be removed?

Reply:
maybe because that is the required part? 🤷